scripts/gluster-block/setup-gluster-block-glusto.yml

```diff
@@ -0,0 +1,392 @@
+    service: name=rpcbind state=started
+
+
+#Set up gluster-block and related packages on servers
+- hosts: gluster_nodes[1:4]
+  tasks:
+  - name: Install dependency packages
+    yum: name={{ item }} state=installed
+    with_items:
+    - git
+    - libtool
+    - libuuid-devel
+    - python-gobject
+    - glusterfs-api-devel
+    - automake
+    - autoconf
+    - gcc
+    - glib2-devel
+    - "*kmod*"
+    - libnl3*
+    - zlib-devel
+    - cmake
+
+  - name: Clone configshell-fb repo
+    git:
+        repo: https://github.com/open-iscsi/configshell-fb
+        dest: /root/configshell-fb
+        refspec: "+refs/heads/master:refs/remotes/origin/master"
+        version: FETCH_HEAD
+
+  - name: Install rtslib
+    command: "python setup.py install chdir=/root/configshell-fb"
+
+  - name: Clone rtslib repo
+    git:
+        repo: https://github.com/open-iscsi/rtslib-fb
+        dest: /root/rtslib-fb
+        refspec: "+refs/heads/master:refs/remotes/origin/master"
+        version: FETCH_HEAD
+
+  - name: Install rtslib
+    command: "python setup.py install chdir=/root/rtslib-fb"
+
+  - name: Clone targetcli-fb repo
+    git:
+        repo: https://github.com/open-iscsi/targetcli-fb
+        dest: /root/targetcli-fb
+        refspec: "+refs/heads/master:refs/remotes/origin/master"
+        version: FETCH_HEAD
+
+  - name: Install targetcli-fb
+    command: "python setup.py install chdir=/root/targetcli-fb"
+
+  - name: Clone tcmu-runner repo
+    git:
+        repo: https://github.com/open-iscsi/tcmu-runner
+        dest: /root/tcmu-runner
+        refspec: "+refs/heads/master:refs/remotes/origin/master"
+        version: FETCH_HEAD
+
+  - name: Install tcmu-runner
+    command: 'chdir=/root/tcmu-runner {{ item }}'
+    with_items:
+    -  sed -i "s/linux\/uio.h/sys\/uio.h/" target_core_user_local.h
+    -  cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr -Dwith-rbd=false -Dwith-qcow=false -DSUPPORT_SYSTEMD=ON -DCMAKE_BUILD_TYPE=Debug
+    -  make install
+
+  - name: Clone gluster-block repo
+    git:
+        repo: https://github.com/gluster/gluster-block
+        dest: /root/gluster-block
+        refspec: "+refs/heads/master:refs/remotes/origin/master"
+        version: FETCH_HEAD
+
+
+  - name: Install gluster-block
+    command: 'chdir=/root/gluster-block {{ item }}'
+    with_items:
+    -  ./autogen.sh
+    -  ./configure
+    -  make
+    -  make install
+
+  - name: Start gluster-block Daemon
+    service: name=gluster-blockd state=started
+
+# Setup packages for gluster-block on client
+- hosts: gluster_nodes[5]
+  tasks:
+  - name: Install dependency packages
+    yum: name={{ item }} state=installed
+    with_items:
+    - iscsi-initiator-utils 
+    - device-mapper-multipath
+
+  - name: Start iscsi Daemon
+    service: name=iscsid state=started
+
+  - name: set multipath
+    command: "modprobe dm_multipath"
+
+  - name: Set multipath
+    command:  "mpathconf --enable"
+
+  - name: append the multipath.conf file
```

Comment:
Use the ansible copy module to copy a file over.

Reply:
multipath.conf file already has some content inside it. I just want to append the new content to this file. I guess using the copy file will replace the old file and its contents and this is not what we want.

Reply:
Can you use `blockinfile` instead, then?

Reply:
@nigelbabu updated the PR